docs/source/upcoming_release_notes/1209-Update_TprTrigger_config_attributes,_add_TCMPL_PV.rst

```diff
@@ -0,0 +1,30 @@
+1209 Update TprTrigger config attributes, add TCMPL PV
```

Comment:
```suggestion
1210 Update TprTrigger config attributes, add TCMPL PV
```
Shouldn't this be 1210 for the pr number instead of the issue?

Reply:
Whoops, yes, fixing...